lnst/Recipes/ENRT/UseVfsMixin.py

```diff
@@ -12,11 +12,16 @@ class UseVfsMixin:
     with VF Device instances. This allows user to interact with the network
     interfaces without additional changes to the code of recipe.
 
+    Mixin provides two boolean parameters:
```

Comment:
As they're not both boolean, I'll update the dosctring.

Reply:
the docstring still says "two boolean parameters"

Reply:
Hmm, so I think I've accidentally originally pushed the code to LNST-project's git tree instead of my fork. Then I pushed the updates into my fork's git, so they're not visible here. I'll fix that.

Reply:
There is no way to change the source branch of PR, I created a new one in https://github.com/LNST-project/lnst/pull/381

Reply:
This is fixed in new PR.